### DISCUSSION TITLE
Export track data like in the Jbrowse 1

### DISCUSSION THREAD

**Discussion by Marie-Lahaye** [GMOD/jbrowse-components#2810] [Q&A] [answered]
Hi,

I was wondering if there was a way to export track data, like in the Jbrowse 1? I remember that we could export track data on the region that we were visualizing.

For example exporting genes in GFF3 format from a specific region:

![image](https://user-images.githubusercontent.com/63525627/158348077-d54bf5e7-8ac0-4371-9887-de99e590f17a.png)

Is there a similar feature with the Jbrowse 2?

Thanks for any answer you have for me !
Marie

**Comment by cmdcolin** [comment, marked as answer]
Indeed, we do not have this feature in jbrowse 2 (yet...)!

If you have gff3tabix tracks, on the command line you can try running

tabix http://host/path/to/your/gff.gz chr1:400-500


This will get a GFF snippet.

We could definitely consider incorporting this to JB2 though

**Reply by Marie-Lahaye** [reply]
Thanks a lot, I'll keep that in mind!
Rather, it was to save users from having to go through the command line. But I guess they will wait until maybe this feature is added, thanks again!

**Reply by jrodiger-lifemine** [reply]
@cmdcolin Our users are also interested in a similar feature to export a selected region in gff or gbk format :)

**Reply by cmdcolin** [reply]
@jrodiger-lifemine thanks for adding the interest, I think this will indeed be a great feature to have. currently tracking this issue here https://github.com/GMOD/jbrowse-components/issues/3094
